src/main/java/com/fvv/bookstore/dao/DvdDAOImpl.java

```diff
@@ -65,8 +65,7 @@ public void addDvd(final Dvd dvd) throws DaoException {
 		List<ShowDvd> dvds = new ArrayList<>();
 		try (
 				Connection conn = ConnectionFactory.getConnection(); 
-				PreparedStatement ps = conn.prepareStatement(
-						SqlQueryEnum.DVD_SELECT_ALL_SHOW.getQuery());
+				PreparedStatement ps = conn.prepareStatement(SqlQueryEnum.DVD_SELECT_ALL_SHOW.getQuery());
```

Comment:
fix javadoc in populateDvdFromDatabase method

Reply:
ok